index.src.html

```diff
@@ -174,79 +174,77 @@
+  <p>This section normatively specifies the API for creating and using scoped credentials. Support for deleting credentials is deliberately omitted; this is expected to be done through platform-specific user interfaces rather than from a script. The basic idea is that the credentials belong to the user and are managed by the browser and underlying platform. Scripts can (with the user&apos;s consent) request the browser to create a new credential for future use by the script's origin. Scripts can also request the user’s permission to perform authentication operations with an existing credential held by the platform. However, all such operations are mediated by the browser and/or platform on the user&apos;s behalf. At no point does the script get access to the credentials themselves; it only gets information about the credentials in the form of objects.</p>
 
   <p>User agents SHOULD only expose this API to callers in <dfn>secure contexts</dfn>, as defined in [[powerful-features]].</p>
 
-  <p>In the future, this API may be integrated into a more general Web API framework for credential management, which is being worked on in the W3C. Such integration will, most likely, create intermediate interface and dictionary types, from which the types in this specification will then inherit. However the experience of the FIDO developer and end user will not be substantially changed by this. In the meantime, this specification is maintained in a more minimal form for ease of review.</p>
-
   <p>The API is defined by the following Web IDL fragment.</p>
 
   <pre class="idl">
-  partial interface Window { 
-      readonly attribute FIDOCredentials fido; 
-  }; 
-   
-  interface FIDOCredentials { 
-      Promise < FIDOCredentialInfo > makeCredential ( 
-          Account                               accountInformation, 
-          sequence < FIDOCredentialParameters > cryptoParameters, 
-          DOMString                             attestationChallenge,
-          optional unsigned long                credentialTimeoutSeconds,
-          optional sequence < Credential >      blacklist,
-          optional FIDOExtensions               credentialExtensions
-      ); 
-   
-      Promise < FIDOAssertion > getAssertion ( 
+  partial interface Window {
+      readonly attribute WebAuthentication webauth;
+  };
+
+  interface WebAuthentication {
+      Promise < ScopedCredentialInfo > makeCredential (
+          Account                                 accountInformation,
+          sequence < ScopedCredentialParameters > cryptoParameters,
+          DOMString                               attestationChallenge,
+          optional unsigned long                  credentialTimeoutSeconds,
+          optional sequence < Credential >        blacklist,
+          optional WebAuthExtensions              credentialExtensions
+      );
+
+      Promise < WebAuthAssertion > getAssertion (
```

Comment:
It seems inconsistent to say WebAuthAssertion but not WebAuthAttestationStatement for example. OTOH Assertion seems too generic. Your thoughts?